source/about/security/mobile-security.rst

```diff
@@ -0,0 +1,48 @@
+Mobile Security
+================
+
+Mattermost mobile is built with a robust security framework to protect user data and prevent unauthorized access. The Mattermost Mobile app's comprehensive security framework ensures that user data remains protected against unauthorized access.
+
+Continuous compliance through :ref:`secure SDLC practices and proactive vulnerability management <deploy/mobile/mobile-security-features:security compliance>` further reinforces the platform’s resilience.
+
+As the mobile app evolves with future enhancements like file sharing restrictions, Mattermost is committed to delivering a secure, reliable mobile experience.
+
+Jailbreak and root detection
+-----------------------------
+
+Jailbreaking or rooting a device disables many built-in security measures, making the device prone to malware and unauthorized access. This protection enables Mattermost administrators to deny access to any mobile device that appears to have been tampered with or is running a non-standard version of the operating system. Additionally, a rooted device may stop checking for software updates and security patches, and it might not be able to install them because the kernel is no longer properly signed. 
+
+By detecting such devices, Mattermost ensures that only secure, uncompromised devices can access sensitive data. When enabled by the system admin, this proactive measure ensures the underlying mobile platform operates reliably and performs the expected kernel and operating system security protections before sending any customer data to the Mattermost application, and minimizes risk in environments where personal device security cannot be guaranteed. Learn more about Mattermost mobile :ref:`jailbreak and root detection <deploy/mobile/mobile-security-features:jailbreak and root detection>`.
+
+Biometric authentication
+------------------------
+
+Native biometric authentication of the user ensures only the authorized device owner is allowed to open the Mattermost application. Biometric authentication uses hardware-level security to significantly enhance data security. 
+
+Biometric authentication significantly enhances security by ensuring that even if a device is lost or stolen, only an authorized user can access the application. It leverages the strengths of hardware-level security, making it far more challenging for attackers to bypass compared to traditional passwords. Biometrics add a layer of security that is both user-friendly and resistant to unauthorized access, especially in a mobile context. Administrators can require the hardware biometric to authenticate the user each time they attempt to access the Mattermost application. This adds another layer of protection to Mattermost customer data, substantially containing the risk from lost or stolen devices. Learn more about Mattermost mobile :ref:`biometric authentication <deploy/mobile/mobile-security-features:biometric authentication>`.
```

Comment:
> Administrators can require the hardware biometric to authenticate the user each time they attempt to access the Mattermost application

This doesn't paint the full picture. We required the biometric in many more cases. We outline all cases here: https://docs.mattermost.com/configure/environment-configuration-settings.html#enable-biometric-authentication